dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ExternalHandleRestRepository.java

```diff
@@ -257,7 +279,7 @@ private boolean matchesAnyOf(String tested, String configPropertyWithPatterns) {
             return false;
         }
 
-        final String pattern = String.join(",", patterns);
+        final String pattern = String.join(";", patterns);
```

Comment:
This is probably not correct:
shortener.post.url.blacklist.regexps = .*://.{1,15}

matchesAnyOf(host, "shortener.post.host.blacklist.regexps")

getArrayProperty splits by `,` you join it with `;` and split again with `;`

Reply:
@vidiecan 
see the hostBlacklist test, I'm hoping that one is testing the config correctly.

getArrayProperty works in two "modes"
```
a=1
a=2
a=3
```
and 
```
a=1,2,3
```
and these should produce the same result `["1", "2", "3"]`.

In v5 we've used `;` for a similar purpose - to define an array of patterns. I'm assuming we are keeping that in v7 for compatibility. 

But what happens if you mix the two approaches?

```
a=1
a=2;3
```
getArrayProperty will load that as `["1", "2;3"]`? If you join that with `,` and then split with `;` you get `["1,2", "3"]`, but I'd say the correct result is `["1", "2", "3"]` and you get that with this changes, at least I hope you do.

Reply:
>shortener.post.url.blacklist.regexps = .*://.{1,15}

I see, it's broken when there's a comma in the regexep